package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@algoan/react-eslint-config",
+  "version": "1.0.0",
+  "description": "Algoan React ESLint configuration",
+  "main": "index.js",
+  "scripts": {
+    "semantic-release": "semantic-release"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/algoan/react-eslint-config.git"
+  },
+  "keywords": [
+    "eslint",
+    "algoan",
+    "react",
+    "typescript",
+    "eslintconfig"
+  ],
+  "author": "Algoan",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/algoan/react-eslint-config/issues"
+  },
+  "homepage": "https://github.com/algoan/react-eslint-config#readme",
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.5.0",
+    "@typescript-eslint/parser": "^4.5.0",
+    "eslint": "^7.11.0",
+    "eslint-config-prettier": "^6.13.0",
+    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^4.2.0"
+  },
+  "peerDependencies": {
+    "prettier": ">= 2",
+    "react": ">= 16.8",
```

Comment:
Since `eslint-plugin-react-hooks` is used,  React version should support hooks ([which were added in version 16.8](https://reactjs.org/docs/hooks-reference.html)).